### PR TITLE
Fix image didn't fill screen area issue

### PR DESCRIPTION
**Pull request by YayunHuang**
Ref: MUP-153

adjust mask position because we add padding for image in: https://github.com/oursky/mockuphone.com/pull/79

https://github.com/user-attachments/assets/0c5bd160-114a-4303-9237-3dd831a614eb

### CODE REVIEW

File: mockup_package/mockup/image_generator.py [hunk 1]
```diff
@@ -190,15 +190,16 @@ def create_mockup_image(self, device_path, device_mask_path_prefix, result_path)
             tmp = self.target_points[0].copy()
             self.target_points[0] = self.target_points[1]
             self.target_points[1] = tmp
+            shifted_points = self.target_points + np.array([PADDING, PADDING])
```
Comment (pkong-ds) [inline mockup_package/mockup/image_generator.py:193]: question: How does adding padding and shifting points not affect the pictures fitting the phone?

e.g. if imageA fits perfectly on a phone (say moto-e) before. after adding padding, does imageA still fits perfectly? If so, how?

Thanks~

Comment (YayunHuang) [reply mockup_package/mockup/image_generator.py:193]: This shows why it become incorrect before this update:
image

in #79 we adjusted the position of the pasted image if `is_mockup_image_at_front`, but forgot to update the other cases

Comment (pkong-ds) [reply mockup_package/mockup/image_generator.py:193]: Thanks yayun! This explanation looks amazing. Can you put it in `mockuphone.com/docs`? 🙇

Comment (YayunHuang) [reply mockup_package/mockup/image_generator.py:193]: @pkong-ds add docs to explain why we add padding when generating mockup ~

**Review by pkong-ds — APPROVED** [review]
Thanks a lot for the great work @YayunHuang 🙏 🙏 🙏

**Pull request merged**